BUILD

```diff
@@ -119,12 +123,52 @@ xcodeproj(
     bazel_path = "./bazelw",
     build_mode = "bazel",
     project_name = "Envoy",
+    scheme_autogeneration_mode = "auto",  # Switch to "all" to generate schemes for all deps
+    schemes = [
+        xcode_schemes.scheme(
+            name = "Async Await App",
+            launch_action = xcode_schemes.launch_action("//examples/swift/async_await:app"),
+        ),
+        xcode_schemes.scheme(
+            name = "Hello World App",
+            launch_action = xcode_schemes.launch_action("//examples/swift/hello_world:app"),
+        ),
+        # TODO(jpsim): Fix Objective-C app support
+        # xcode_schemes.scheme(
+        #     name = "Hello World App (ObjC)",
+        #     launch_action = xcode_schemes.launch_action("//examples/objective-c/hello_world:app"),
+        # ),
+        xcode_schemes.scheme(
+            name = "Baseline App",
+            launch_action = xcode_schemes.launch_action("//test/swift/apps/baseline:app"),
+        ),
+        xcode_schemes.scheme(
+            name = "Experimental App",
+            launch_action = xcode_schemes.launch_action("//test/swift/apps/experimental:app"),
+        ),
+        xcode_schemes.scheme(
+            name = "Swift Library",
+            build_action = xcode_schemes.build_action(["//library/swift:ios_lib"]),
+        ),
+        xcode_schemes.scheme(
+            name = "iOS Tests",
+            test_action = xcode_schemes.test_action([
+                "//experimental/swift:quic_stream_test.__internal__.__test_bundle",
```

Comment:
would be nice to not have to rely on this implementation detail

Reply:
Agreed. https://github.com/buildbuddy-io/rules_xcodeproj/issues/215